Gemfile

```diff
@@ -13,11 +13,13 @@ gem 'devise-async'
 gem 'dotenv-rails'
 gem 'enumerize'
 gem 'font_assets', github: "ericallam/font_assets", branch: 'master'
+gem 'foreigner'
```

Comment:
Rails 4.2 has support to FK https://github.com/Codeminer42/cm42-central/pull/32
just need to find a way to solve the breaking test